common/lib/xmodule/xmodule/tabs.py

```diff
@@ -851,48 +271,23 @@ def get_tab_by_id(tab_list, tab_id):
         return next((tab for tab in tab_list if tab.tab_id == tab_id), None)
 
     @staticmethod
-    def iterate_displayable(
-            course,
-            settings,
-            is_user_authenticated=True,
-            is_user_staff=True,
-            is_user_enrolled=False
-    ):
+    def iterate_displayable(course, user=None, inline_collections=True):
         """
         Generator method for iterating through all tabs that can be displayed for the given course and
         the given user with the provided access settings.
         """
         for tab in course.tabs:
-            if tab.can_display(
-                    course, settings, is_user_authenticated, is_user_staff, is_user_enrolled
-            ) and (not tab.is_hideable or not tab.is_hidden):
+            if tab.is_enabled(course, user=user) and (not user or not tab.is_hideable or not tab.is_hidden):
```

Comment:
This boolean is confusing. Would it read better as `not (user and tab.is_hideable and tab.is_hidden)`?